tcs/tcs_listener/tcs_listener.py

```diff
@@ -119,21 +120,17 @@ def _process_request(self, input_json_str):
             return response
 
     def render_GET(self, request):
-        response = {}
-        response['error'] = {}
-        response['error']['code'] = WorkorderError.INVALID_PARAMETER_FORMAT_OR_VALUE
-        response['error']['message'] = 'Only POST request is supported'
+        response = utility.create_error_response(WorkorderError.INVALID_PARAMETER_FORMAT_OR_VALUE, "0", \
```

Comment:
Just so you know, in Python, a "\" at the end of the line is not needed to continue function parameters on the next line.
 Python knows there are more parameters because the "(" is not closed yet with ")". In other words, you can leave out the "\" at the end of the line (here and elsewhere).

Reply:
thanks for the info, i will remove it.